webpack.config.js

```diff
@@ -57,6 +65,9 @@ module.exports = ( env ) => {
       extensions: [ '.js', '.ts', '.tsx' ]
     },
 
+    devServer: {
```

Comment:
Because **localhost** is **127.0.0.1** and it is useless when you need to test from e.g. VM or when you are running it in VM and want to test from outside.

Typical use case is testing in IEs.